tests/components/daikin/test_config_flow.py

```diff
@@ -0,0 +1,94 @@
+# flake8: noqa pylint: skip-file
```

Comment:
It's better to just disable the things we don't care about like the warning about redefined variable when using the pytest fixtures. Then lazy tox eg will still catch things that we should follow.